Geometry_Engine/Create/Vector.cs

```diff
@@ -31,45 +34,61 @@ public static partial class Create
         /**** Public Methods                            ****/
         /***************************************************/
 
+        [Description("Creates a Vector based on its coordinates.")]
+        [InputFromProperty("x")]
+        [InputFromProperty("y")]
+        [InputFromProperty("z")]
+        [Output("vec", "The created Vector.")]
```

Comment:
(applies to all similar methods)
```suggestion
        [Output("vector", "The created Vector.")]
```